webserver/src/components/GlycanDetail/GlycanDetail.tsx

```diff
@@ -14,7 +16,7 @@ export default function GlycanDetail(props) {
        
     }
 
-    const ref = useCallback((node) => {
+    const ref = useCallback((node: any) => {
```

Comment:
Is the use of `any` type for SVG node unavoidable here? I can see how `any` is well justified due to typescript potentially complaining about SVG node, but at the same time the dogma appears to be to avoid the use of `any` whenever possible.